media-tv/kodi/kodi-9999.ebuild

```diff
@@ -133,6 +133,11 @@ DEPEND="${COMMON_DEPEND}
 # generated addons package.  #488118
 [[ ${PV} == "9999" ]] && DEPEND+=" virtual/jre"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-9999-no-arm-flags.patch #400618887
+	"${FILESDIR}"/${PN}-9999-texturepacker.patch
+)
```

Comment:
Other ebuilds don't have it as a local variable. I think this approach complies with EAPI=6 and follows the examples I've seen. Are you sure it needs to move?